src/brackets.js

```diff
@@ -67,6 +67,7 @@ define(function (require, exports, module) {
         DocumentManager         = require("document/DocumentManager"),
         EditorManager           = require("editor/EditorManager"),
         JSUtils                 = require("language/JSUtils"),
+        XMLUtils                = require("language/XMLUtils"),
```

Comment:
Also, this causes a JSHint error (see the Travis failure) because the var is unused. You'll want to add this to the section on line 98 instead.